helpers/inject_account_id.py

```diff
@@ -0,0 +1,76 @@
+import pathlib
+import boto3
+from colorama import Fore
+import os
+import yaml
+
+def inject_account_id():
+    # Updates policy files:
+    # If the policy file contains `{your_account_id}`, replaces `{your_account_id}` with accound ID extracted from STS so policy can be executed
```

Comment:
Custodian already has this function where it will replace the variable {account} in policies with the account id the policy is running in.

Reply:
@jtroberts83 Yesss! I was hoping you would say that! Can you point me to the documentation for this? Because I somehow missed it on my own 😭

Reply:
Link is here: https://cloudcustodian.io/docs/aws/lambda.html?highlight=account_id#periodic-function